### DISCUSSION TITLE
How can I prevent the first story from being shown twice?

### DISCUSSION THREAD

**Discussion by andyford** [storybookjs/storybook#24700] [Help] [answered]
It never really bothered me that the first story was repeated, but now I have a group of radio buttons in a story. So when the story is repeated, they act as one large group since all the radio inputs have the same name attribute.

<img width="820" alt="Screenshot 2023-11-03 at 14 33 29" src="https://github.com/storybookjs/storybook/assets/49811/71a1f4cc-781c-475a-947b-3a84f62859c0">

I'm using storybook 7.5 for react/typescript. Here's the content of my `RadioCardGroup.stories.tsx` file:
```TypeScript
import { Meta, StoryFn } from '@storybook/react';

import { RadioCard } from '../RadioCard';
import { RadioCardGroup, RadioCardGroupProps } from './RadioCardGroup';

export default {
  title: 'Components/Form/RadioCardGroup',
  component: RadioCardGroup,
} as Meta;

const Template: StoryFn<RadioCardGroupProps> = (args) => {
  return (
    <RadioCardGroup {...args}>
      <RadioCard name="demo-group-Default" value="1" id="demo-group-Default--1">
        One
      </RadioCard>

      <RadioCard name="demo-group-Default" value="2" id="demo-group-Default--2">
        Two
      </RadioCard>

      <RadioCard name="demo-group-Default" value="3" id="demo-group-Default--3">
        Three
      </RadioCard>
    </RadioCardGroup>
  );
};

export const Default = Template.bind({});
Default.storyName = 'default';
```

**Comment by xsu1010** [comment, marked as answer]
If you're using custom `.mdx` files to generate your documentation (e.g. you have a `RadioCardGroup.mdx` file or you have a custom template for your autodocs), you can pass the [`includePrimary` parameter](https://storybook.js.org/docs/react/api/doc-block-stories#includeprimary) to your `<Stories/>` docblock to omit the Primary story:

```mdx
{/* RadioCardGroup.mdx */}

{/* ... */}

<Stories includePrimary={false} />
```

If you don't have anything like that set up, you'll need to [customize your autodocs.](https://storybook.js.org/docs/react/writing-docs/autodocs#write-a-custom-template) From there, you can then disable the primary story from showing up.
Something like this will do the trick:

```tsx
// .storybook/preview.tsx

// Replace your-framework with the framework you are using (e.g., react, vue3)
import { Preview } from '@storybook/your-framework';

import { Title, Subtitle, Description, Primary, Controls, Stories } from '@storybook/blocks';

const preview: Preview = {
  parameters: {
    actions: { argTypesRegex: '^on[A-Z].*' },
    controls: {
      matchers: {
        color: /(background|color)$/i,
        date: /Date$/,
      },
    },
    docs: {
      page: () => (
        <>
          <Title />
          <Subtitle />
          <Description />
          <Primary />
          <Controls />
          <Stories includePrimary={false}/>
        </>
      ),
    },
  },
};

export default preview;
```

**Reply by joevaugh4n** [reply]
Thank you for jumping in here, @xsu1010 – we super appreciate it!

**Reply by andyford** [reply]
Hey sorry for the delayed reply and thanks for the answer. I already had a super similar setup in preview.tsx so all I needed to do was add `includePrimary={false}` to `<Stories />` and that did the trick! Thank you!

**Reply by joevaugh4n** [reply]
Amazing, really glad it worked for you @andyford!

Thanks again, @xsu1010!
